packages/datadog_flutter_plugin/android/src/main/kotlin/com/datadoghq/flutter/DatadogRumPlugin.kt

```diff
@@ -341,6 +347,19 @@ class DatadogRumPlugin(
         result.success(null)
     }
 
+    private fun sessionStarted(sessionId: String, isDiscarded: Boolean) {
+        val handler = Handler(Looper.getMainLooper())
+        handler.post {
+            channel.invokeMethod(
+                "rumSessionStarted",
+                mapOf(
+                    "sessionId" to sessionId,
+                    "sampled" to isDiscarded,
```

Comment:
shouldn't it be `!isDiscared` here? because if `isDiscarded = true`, it means that session is sampled out, meaning that `sampled` should be `false`.

Reply:
Yeah, I'm going to revisit the name of this variable. `sampled` here is supposed to mean "sampled out" which means `isDiscarded = true` but considering I actually messed it up twice while unit testing, I should have realized it's a bad name.